exporter/alibabacloudlogserviceexporter/metricsdata_to_logservice.go

```diff
@@ -194,10 +177,18 @@ func numberMetricsToLogs(name string, data pdata.NumberDataPointSlice, defaultLa
 			labels.Append(k, v)
 			return true
 		})
-		logs = append(logs, newMetricLogFromRaw(name,
-			labels,
-			int64(dataPoint.Timestamp()),
-			dataPoint.Value()))
+		switch dataPoint.Type() {
+		case pdata.MetricValueTypeInt:
+			logs = append(logs, newMetricLogFromRaw(name,
+				labels,
+				int64(dataPoint.Timestamp()),
+				float64(dataPoint.IntVal())))
```

Comment:
The indentation is a bit strange, since the args are for `newMetricLogFromRaw` can you try to change it?

Reply:
Updated the code, please take a look.